stubs/cpp/package/conanfile.py

```diff
@@ -0,0 +1,35 @@
+from conans import ConanFile, CMake
+
+
+class StellarstationapiConan(ConanFile):
```

Comment:
Cases of class name looks not usual. Is this intentional?

Reply:
It was automatically generated by conan, but seems like we can rename it.